### PR TITLE
added youtube and google drive sharable video services

### PR DESCRIPTION
**Pull request by JagadishPujari**
[JIRA LINK](https://project-sunbird.atlassian.net/browse/SB-10218)

### CODE REVIEW

File: app/scripts/framework/service/meta-service.js [hunk 1]
```diff
@@ -162,6 +162,9 @@ org.ekstep.services.metaService = new (org.ekstep.services.iService.extend({
 	},
 
 	getVideoLicense: function (data, callback) {
-		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/license/validate'), data, this.requestHeaders, callback)
+		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/validate?field=license'), data, this.requestHeaders, callback)
```
Comment (pallakartheekreddy) [inline app/scripts/framework/service/meta-service.js]: You can pass `field` also in for function same function will be useful later if they add one more fieldType

File: app/scripts/framework/service/meta-service.js [hunk 2]
```diff
@@ -161,7 +161,10 @@ org.ekstep.services.metaService = new (org.ekstep.services.iService.extend({
 		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('configEndPoint', '/data') + this.getConfig('configurationUrl', '/v1/form/read'), data, this.requestHeaders, callback)
 	},
 
-	getVideoLicense: function (data, callback) {
-		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/license/validate'), data, this.requestHeaders, callback)
+	getVideoLicense: function (data, field,callback) {
```
Comment (vinukumar-vs) [inline app/scripts/framework/service/meta-service.js]: indentation missing. These kind of indentation issues are not acceptable. I am seeing frequently. 
data, field,callback

File: app/scripts/framework/service/meta-service.js [hunk 3]
```diff
@@ -161,7 +161,10 @@ org.ekstep.services.metaService = new (org.ekstep.services.iService.extend({
...
+		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('videoLicenseURL', '/asset/v3/validate?field=' + field), data, this.requestHeaders, callback)
+	},
+	getDriveVideoFileSize: function (data, field, callback) {
```
Comment (vinukumar-vs) [inline app/scripts/framework/service/meta-service.js]: There is no difference between getVideoLicense & getDriveVideoFileSize. Why do you need this new API

File: app/scripts/framework/service/meta-service.js (hunk 1 above)
Comment (JagadishPujari) [reply app/scripts/framework/service/meta-service.js]: Yes, changes are done and pushed

File: app/scripts/framework/service/meta-service.js (hunk 2 above)
Comment (JagadishPujari) [reply app/scripts/framework/service/meta-service.js]: indentation is done and pushed

File: app/scripts/framework/service/meta-service.js (hunk 3 above)
Comment (JagadishPujari) [reply app/scripts/framework/service/meta-service.js]: Removed getDriveVideoFileSize method

**Pull request merged**